JavaScript/9-bigint.js

```diff
@@ -51,7 +51,13 @@ console.log();
 const array1 = [-2, 7, 1n, 3, -2n, 8n, 5, -4n];
 console.log(array1);
 console.log(array1.sort());
-console.log(array1.sort((a, b) => BigInt(a) - BigInt(b)));
+console.log(array1.sort((a, b) => {
+  const biA = BigInt(a);
+  const biB = BigInt(b);
+  if (biA < biB) return -1;
+  if (biA > biB) return 1;
+  return 0;
+}));
```

Comment:
```suggestion
console.log(array1.sort((a, b) => Number(BigInt(a) - BigInt(b)));
```

Reply:
I also thought of this, but it won't work when there will really huge BigInt values in the array.

Changed in [86a18cf](https://github.com/HowProgrammingWorks/DataTypes/pull/137/commits/86a18cf974d1a816a056fc261fb2c8179e70937f)